olive/model/handler/pytorch.py

```diff
@@ -217,9 +217,8 @@ def _load_mlflow_model(self):
                     raise ValueError(
                         "Unsupported MLFlow model flavor. Currently only support hftransformersv2/hftransformers."
                     )
-
-            model_loader = huggingface_model_loader(hf_pretrained_class)
-            loaded_model = model_loader(tmp_dir)
+            loading_args = self.hf_config.get_loading_args_from_pretrained() if self.hf_config else {}
+            loaded_model = load_hf_model_from_model_class(hf_pretrained_class, tmp_dir, **loading_args)
```

Comment:
what is `tmp_dir` used for?

Reply:
It is the folder of intermedia model, used for huggingface to load.